soc/xtensa/intel_adsp/ace/Kconfig.defconfig.series

```diff
@@ -47,10 +47,13 @@ config XTENSA_TIMER_ID
 
 config SYS_CLOCK_HW_CYCLES_PER_SEC
 	default 400000000 if XTENSA_TIMER
-	default 19200000 if INTEL_ADSP_TIMER
+	default 38400000 if INTEL_ADSP_TIMER
 
 config SYS_CLOCK_TICKS_PER_SEC
-	default 50000
+	default 12000
+
+config XTENSA_CCOUNT_HZ
+	default 400000000
```

Comment:
Why 400MHz?

Reply:
This comes from `platform_cpu_freq` in `src/platform/meteorlake/lib/clk.c` under SOF, which defines the CPU frequency (which is also the CCOUNT frequency).

I have changed this and the timer rate (XTENSA_TIMER above).

Reply:
> This comes from platform_cpu_freq in src/platform/meteorlake/lib/clk.c under SOF,

It wouldn't hurt having this in the commit message at least, maybe as a comment even.

Reply:
Hm... but with 393.216MHz, it's no longer coming from SOF (at least grep didn't find it).

Reply:
Value in `platform_cpu_freq` is probably copy-paste error.